[CIAPP] Document support for logs in CI Visibility junit XML upload (#13680)
DataDog/documentation: content/en/continuous_integration/setup_tests/junit_upload.md

**Example**: `team:backend`<br/>
**Note**: Tags specified using `--tags` and with the `DD_TAGS` environment variable are merged. If the same key appears in both `--tags` and `DD_TAGS`, the value in the environment variable `DD_TAGS` takes precedence.

`--logs` **(beta)**

> : Enable forwarding content from the XML reports as [Logs][7]. The content inside `<system-out>`, `<system-err>`, `<failure>` will be loaded. Logs from elements inside a `<testcase>` are automatically connected to test.<br/>

Reviewer: ```suggestion
: Enable forwarding content from the XML reports as [Logs][7]. The content inside `<system-out>`, `<system-err>`, `<failure>` will be sent to the Logs system. Logs from elements inside a `<testcase>` are automatically linked to the test.<br/>
```

Author: I used the word connected because of https://docs.datadoghq.com/tracing/connect_logs_and_traces/ but good suggestions for the others I'll make the changes

> **Default**: `false`<br/>

Reviewer 2: ```suggestion
**Environment variable**: `DD_CIVISIBILITY_LOGS_ENABLED`<br/>
**Default**: `false`<br/>
```

> **Note**: Logs are billed separately from CI Visibility.

Reviewer 2: cc @KickingTheTV

**Example**: `team:backend`<br/>
**Note**: Tags specified using `--tags` and with the `DD_TAGS` environment variable are merged. If the same key appears in both `--tags` and `DD_TAGS`, the value in the environment variable `DD_TAGS` takes precedence.

`--logs` **(beta)**

> : Enable forwarding content from the XML reports as [Logs][7]. The content inside `<system-out>`, `<system-err>`, `<failure>` is collected as logs. Logs from elements inside a `<testcase>` are automatically connected to the test.<br/>

Reviewer 3: ```suggestion
: Enable forwarding content from the XML reports as [Logs][7]. The content inside `<system-out>`, `<system-err>`, and `<failure>` is collected as logs. Logs from elements inside a `<testcase>` are automatically connected to the test.<br/>
```
```suggestion
: Enable forwarding content from the XML reports as [Logs][7]. The content inside `<system-out>`, `<system-err>`, `<failure>` is collected as logs. Logs from elements inside a `<testcase>` are automatically connected to the test.<br/>
```

> : Enable forwarding content from the XML reports as [Logs][7]. The content inside `<system-out>`, `<system-err>`, `<failure>` is collected as logs. Logs from elements inside a `<testcase>` are automatically connected to the test.<br/>

Reviewer 3: ```suggestion
: Enable forwarding content from the XML reports as [Logs][7]. The content inside `<system-out>`, `<system-err>`, and `<failure>` is collected as logs. Logs from elements inside a `<testcase>` are automatically connected to the test.<br/>
```
```suggestion
: Enable forwarding content from the XML reports as [Logs][7]. The content inside `<system-out>`, `<system-err>`, `<failure>` is collected as logs. Logs from elements inside a `<testcase>` are automatically connected to the test.<br/>
```

**Default**: `datadoghq.com`<br/>
**Selected site**: {{< region-param key="dd_site" code="true" >}}

> `DD_CIVISIBILITY_LOGS_ENABLED` **(beta)**
> : Same as the `--logs` flag: Enable forwarding content from the XML reports as [Logs][7]. The content inside `<system-out>`, `<system-err>`, `<failure>` is collected as logs. Logs from elements inside a `<testcase>` are automatically connected to the test.<br/>
> **Default**: `false`
> 

Reviewer 2: ```suggestion
```
